JAMClock.py

```diff
@@ -185,7 +188,7 @@ def __init__(self):
 		self.connect("embedded", self.embed_event)
 
 	       	os.putenv('SDL_WINDOWID', str(self.eventbox.socket.get_id()))
-		gobject.idle_add(self.get_run_game)
+		GObject.idle_add(self.get_run_game)
```

Comment:
GObject.idle_add is an alias for GLib.idle_add, and we get depreciation warnings for it.